types/api/commands.d.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+interface CommandResponse {
+    goto_location?: string;
+    trigger_id?: string;
+
+    // Defined in the server, but not used in the client
+    // response_type?: string;
+    // text?: string;
+    // username?: string;
+    // channel_id?: string;
+    // icon_url?: string;
+    // type?: string;
+    // props?: string;
+    // skip_slack_parsing?: boolean;
+    // attachments?: any[];
+    // extra_responses?: any[];
```

Comment:
Probably not. I left it here in case we saw anything that ring a bell, but we can remove this whole comment, and add the needed parts in the future if needed.